server/pkg/sort/direction.go

```diff
@@ -0,0 +1,28 @@
+package sort
```

Comment:
too abstract packaga name. also it conflicts with `sort` package in golang. DIrection type shold be implemented in `item` package because it is currently only used in `item` package.